packages/core/src/login/webview/vue/amazonq/backend_amazonq.ts

```diff
@@ -113,4 +113,8 @@ export class AmazonQLoginWebview extends CommonAuthWebview {
     ): Promise<AuthError | undefined> {
         throw new Error('Method not implemented.')
     }
+
+    async quitLoginScreen() {
+        await vscode.commands.executeCommand('setContext', 'aws.amazonq.showAuthView', false)
```

Comment:
I think we shouldn't allow users to quit Amazon Q auth. If they aren't connected, it should always display the login screen. It should never be possible to display the login screen if they are connected.
Maybe we should gather more thoughts on this.